### PR TITLE
Vignette: data input

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

File: vignettes/getting_started.Rmd [hunk 1]
```diff
@@ -150,7 +150,19 @@ str(inputDat)
...
-#### readDeamonXlsx
+#### ZKDeamon
+
+ZKDeamon is a program used by IQB that can be found in the IQB internal folders (i:). After installing you can get the meta data and the items from your specific study using information stored in the IQB Databases (DB2/DB3/DB4). Alternatively you can get the meta data from an SPSS file (see below). You can produce Excel files, which are expected to have the following sheets: “units”, “subunits”, “values”, “unitrecoding”, “sav-files”, “params”, “aggregate-missings”, “itemproperties”, “propertylabels”, “booklets”, and “blocks”. 
+
+The function `readDaemonXlsx()` reads the Excel file into R and will produce a warning if any sheets are missing. It needs one character string containing path, name and extension of the Excel file (.xlsx) produced by ZKDeamon. It returns a list of data frames containing information that is required by the data preparation functions. `inputList` shows an example of this list.
+
+
+```{r readDeamonXlsx, }
+readDaemonXlsx(system.file("extdata", "inputList.xlsx", package = "eatPrep"))
```
Comment (grewered) [inline vignettes/getting_started.Rmd:161]: this produces a very long output. Is that ok?

File: vignettes/getting_started.Rmd [hunk 2]
```diff
@@ -150,7 +150,19 @@ str(inputDat)
 
 To do this, we first need to get the information from our IQB data bank via ZKDeamon, or alternatively via SPSS. 
 
-#### readDeamonXlsx
+#### ZKDeamon
+
+ZKDeamon is a program used by IQB that can be found in the IQB internal folders (i:). After installing you can get the meta data and the items from your specific study using information stored in the IQB Databases (DB2/DB3/DB4). Alternatively you can get the meta data from an SPSS file (see below). You can produce Excel files, which are expected to have the following sheets: “units”, “subunits”, “values”, “unitrecoding”, “sav-files”, “params”, “aggregate-missings”, “itemproperties”, “propertylabels”, “booklets”, and “blocks”.
```
Comment (grewered) [inline vignettes/getting_started.Rmd:155]: I didn't add how to install ZKDeamon or how to use it. I can imagine a link to the program would be nice, but as it's internal, I wasn't sure what to write. any suggestions?

File: vignettes/getting_started.Rmd (hunk 1 above)
Comment (sachseka) [reply vignettes/getting_started.Rmd:161]: indeed, maybe like this
inpustList <- readDaemonXlsx(system.file("extdata", "inputList.xlsx", package = "eatPrep"))
plus (if necessary str(inputList)
it is less disturbing the reading flow?

**Review by sachseka — APPROVED** [review]
LGTM (but maybe change the R code to the suggestion in my comment)

**Comment by grewered** [comment]
@sachseka I made some changes and wrote some more. Can you look over it?

**Comment by grewered** [comment]
@sachseka I've finished the draft for now.

**Pull request merged**